src/Qowaiv.Validation.DataAnnotations/Qowaiv.Validation.DataAnnotations.csproj

```diff
@@ -60,9 +64,7 @@ v0.0.4
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Update="QowaivValidationMessages.nl.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-    </EmbeddedResource>
+    <EmbeddedResource Update="QowaivValidationMessages.nl.resx" />
```

Comment:
You don't need the generator for the dutch messages anymore?

Reply:
Never needed.